packages/webdriverjs/tests/axe-webdriverjs.spec.ts

```diff
@@ -91,24 +100,30 @@ describe('@axe-core/webdriverjs', () => {
       assert.isDefined(results.testEnvironment.windowWidth);
       assert.isDefined(results.testRunner.name);
       assert.isDefined(results.toolOptions.reporter);
-      assert.equal(results.url, `${addr}/index.html`);
+      assert.equal(results.url, `${addr}/external/index.html`);
     });
 
     it('properly isolates the call to axe.finishRun', async () => {
       let err;
       await driver.get(`${addr}/external/isolated-finish.html`);
+      const title = await driver.getTitle();
       try {
         await new AxeBuilder(driver).analyze();
       } catch (e) {
         err = e;
       }
+
+      assert.notEqual(title, 'Error');
       assert.isUndefined(err);
     });
 
     it('throws if axe errors out on the top window', done => {
       driver
         .get(`${addr}/external/crash.html`)
         .then(() => {
+          driver.getTitle().then(res => {
+            assert.notEqual(res, 'Error');
+          });
```

Comment:
This is a race condition since the  code isn't waiting for the response  before going to the analyze.